src/index.css

```diff
@@ -2,6 +2,10 @@
 @tailwind components;
 @tailwind utilities;
 
+.dark {
+  color-scheme: dark;
+}
```

Comment:
Can you explain what you done here?

Reply:
This thing fixed the problem with the scrollbar. There was a problem with the corners of the scrollbar, when you changed the theme, the corners remained white.